lib/modules/platform/gitea/index.md

```diff
@@ -1,25 +1,38 @@
 # Gitea and Forgejo
 
-Renovate supports [Gitea](https://gitea.io) and the new fork [Forgejo](https://forgejo.org).
+Renovate supports [Gitea](https://gitea.io) and the [Forgejo](https://forgejo.org) fork.
 Forgejo exists since `v1.18.0` and is currently fully compatible to Gitea.
 
 ## Authentication
 
-First, [create a Personal Access Token](https://docs.gitea.io/en-us/api-usage/#authentication) for the bot account.
-Let Renovate use your PAT by doing _one_ of the following:
+First, [create a Personal Access Token (PAT)](https://docs.gitea.io/en-us/api-usage/#authentication) for the bot account.
+Then let Renovate use your PAT by doing _one_ of the following:
 
 - Set your PAT as a `token` in your `config.js` file
 - Set your PAT as an environment variable `RENOVATE_TOKEN`
 - Set your PAT when you run Renovate in the CLI with `--token=`
 
-Remember to set `platform=gitea` somewhere in your Renovate config file.
-Either the account should have full name and email address set to allow Renovate to estabilish Git identity, or the `gitAuthor` config option should be set.
+You must set `platform=gitea` in your Renovate config file.
+
+Either the account should have full name and email address set to allow Renovate to establish Git identity, or the `gitAuthor` config option should be set.
```

Comment:
> Either the account should have full name and email address set to allow Renovate to establish Git identity, or the `gitAuthor` config option should be set.

Reading the current docs it's not clear to me what's the best way to authenticate to the Gitea platform. It looks like there are three ways:

- full name and email address
- `gitAuthor` config option
- User scopes (need Gitea `v1.19`)

If possible, the maintainers should decide on _one method_, that we will then recommend.

Having one method makes it easier for me to restructure the document. Right now it's a confusing mess of config methods... 😄 

Let me know what you think maintainers. 😉

Reply:
I think you need 1+3 combined

Reply:
I'm thinking of splitting things up into:

```markdown
## Authentication
### Basic requirements // put things that you always need here
### Full token // the "old way"
### User scopes // the new "recommended" way that relies on Gitea v1.19+
```

But I don't know what each heading should contain.